arrows/vxl/hashed_image_classifier.cxx

```diff
@@ -0,0 +1,518 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "hashed_image_classifier.h"
+
+#include <vil/vil_plane.h>
+
+#include <boost/lexical_cast.hpp>
```

Comment:
Is there an alternative to using boost here? I would really prefer reducing boost rather than increasing.